server/src/main/java/org/apache/druid/catalog/model/Columns.java

```diff
@@ -34,58 +35,72 @@ public class Columns
 {
   public static final String TIME_COLUMN = "__time";
 
-  public static final String VARCHAR = "VARCHAR";
-  public static final String BIGINT = "BIGINT";
-  public static final String FLOAT = "FLOAT";
-  public static final String DOUBLE = "DOUBLE";
-  public static final String TIMESTAMP = "TIMESTAMP";
+  public static final String STRING = ValueType.STRING.name();
+  public static final String LONG = ValueType.LONG.name();
+  public static final String FLOAT = ValueType.FLOAT.name();
+  public static final String DOUBLE = ValueType.DOUBLE.name();
+
+  public static final String SQL_VARCHAR = "VARCHAR";
+  public static final String SQL_BIGINT = "BIGINT";
+  public static final String SQL_FLOAT = "FLOAT";
+  public static final String SQL_DOUBLE = "DOUBLE";
+  public static final String SQL_TIMESTAMP = "TIMESTAMP";
 
   public static final Set<String> NUMERIC_TYPES =
-      ImmutableSet.of(BIGINT, FLOAT, DOUBLE);
+      ImmutableSet.of(LONG, FLOAT, DOUBLE);
   public static final Set<String> SCALAR_TYPES =
-      ImmutableSet.of(TIMESTAMP, VARCHAR, BIGINT, FLOAT, DOUBLE);
+      ImmutableSet.of(STRING, LONG, FLOAT, DOUBLE);
 
   public static final Map<String, ColumnType> SQL_TO_DRUID_TYPES =
       new ImmutableMap.Builder<String, ColumnType>()
-        .put(TIMESTAMP, ColumnType.LONG)
-        .put(BIGINT, ColumnType.LONG)
-        .put(FLOAT, ColumnType.FLOAT)
-        .put(DOUBLE, ColumnType.DOUBLE)
-        .put(VARCHAR, ColumnType.STRING)
-        .build();
+          .put(SQL_BIGINT, ColumnType.LONG)
+          .put(SQL_VARCHAR, ColumnType.STRING)
```

Comment:
Do you mean to remove mappings for `SQL_FLOAT` and `SQL_DOUBLE` and `TIMESTAMP` here?

Reply:
This part wasn't being used actually right now, removed it